cartool/main.m

```diff
@@ -185,8 +215,19 @@ void exportCarFileAtPath(NSString * carPath, NSString *outputDirectoryPath)
 		NSMutableArray *images = getImagesArray(catalog, key);
 		for( CUINamedImage *image in images )
 		{
-			if( CGSizeEqualToSize(image.size, CGSizeZero) )
+            if ( ![image respondsToSelector:@selector(size)]) {
```

Comment:
This `if` statement is kinda sloppy — I wish I could do something like `if ( ![image isKindOfClass:[CGPDFDocument class]])`, but of course `CGPDFDocument` isn't a class and I don't think there's any way to check the struct type at runtime. Wonder if you know of anything better.